src/utils/utils.ts

```diff
@@ -165,12 +166,7 @@ export function findReplace(str: string, re: RegExp, fn: (args) => string) {
   });
 }
 export function path2RegexPattern(path: string) {
```

Comment:
can we just get rid of that function and use simplifiedRegex?

Reply:
Ill have a look

Do we want to change back to using split/join likei was before too?
Might work if I give it another try, dunno if it willbe faster, just need to do the windows paths before anything else

Reply:
I currently replaced all `simplifiedRegex` to `path2RegexPattern`
`simplifiedRegex` does tell me anythin, while `path2RegexPattern` is more spesific in what it does
`path2RegexPattern` is only located under utils now.